ParksAndGardens/GreenAreaRecord/doc/spec.md

```diff
@@ -0,0 +1,103 @@
+# GreenAreaRecord
+
+## Description
+
+This entity contains a harmonised description of the conditions recorded on a particular area or point inside a green area (flower bed, garden, etc.).
+This entity type has been inspired by the `AgriParcelRecord` entity type defined by the GSMA Harmonized Data Models. 
+
+## Data Model
+
+A JSON Schema corresponding to this data model can be found {{add link to JSON Schema}}
+
++ `id` : Unique identifier. 
+
++ `type` : Entity type. It must be equal to `GreenAreaRecord`.
+
++ `dateModified` : Last update timestamp of this entity.
+    + Attribute type: [DateTime](https://schema.org/DateTime)
+    + Optional
+
++ `dateCreated` : Entity's creation timestamp.
+    + Attribute type: [DateTime](https://schema.org/DateTime)
+    + Optional
+    
++ `source` : A sequence of characters giving the source of the entity data.
+    + Attribute type: [Text](https://schema.org/Text) or [URL](https://schema.org/URL)
+    + Optional
+
++ `location` : Location of the area concerned by this record and represented by a GeoJSON geometry. 
+    + Attribute type: `geo:json`.
+    + Normative References: [https://tools.ietf.org/html/rfc7946](https://tools.ietf.org/html/rfc7946)
+    + Mandatory
+    
++ `dateObserved` : The date and time of this observation in ISO8601 UTCformat.
+    + Attribute type: [DateTime](https://schema.org/DateTime). 
+    + Mandatory
+
++ `soilTemperature` : The observed soil temperature in degrees centigrade. 
```

Comment:
It seems "centigrade" is an old fashion way of refering to "Celsius degrees", anyhow I would suggest putting the type of degrees before the noun such as: "Celsius degrees" ;)